OEM/Z370 AORUS Gaming 7/config.plist

```diff
@@ -78,15 +78,15 @@
         <string>DMAR</string>
       </dict>
       <dict>
-        <key>#TableId</key>
+        <key>TableId</key>
```

Comment:
If we keep the # at the beginning, the file won't open in Clover Configurator. It's not a big deal, but are you sure it's necessary to keep the `#`?

Reply:
You can change `#` to `_`, but using a Clover Configurator it's not good idea!

Reply:
why not? (again, new to this, sorry!)

Reply:
CC (Clover Configurator) not maintained by Clover developers. In same cases CC can corrupt config file. I'm using a PlistEdit Pro in macOS and Sublime Text on Windows. To work with base64 and another binary stuff I use https://goo.gl/tz7scz

Reply:
Or you can change `#TableId` to `Comment` its will be great to know what is goes to drop.